### DISCUSSION TITLE
Next Image not working in Storybook

### DISCUSSION THREAD

**Discussion by bmarshall511** [RyanClementsHax/storybook-addon-next#76] [Q&A] [answered]
See https://stackoverflow.com/questions/72003497/next-js-storybook-remote-images-not-working

Also, relative images don't seem to be working. For instance, have an image in public/images/image1.jpg and calling it like so:

```
<Image
  src="/images/image1.jpg"
  layout="fill"
  objectFit="cover"
  objectPosition="center"
  alt=""
/>
```

But results in a 404.

**Comment by RyanClementsHax** [comment, marked as answer]
Thanks for bringing this up!

This issue is being tracked on #72
